samples/default-field-value/README.md

```diff
@@ -0,0 +1,50 @@
+## Default field values
+
+This UI Extension uses configuration parameters to set a default value for a field.
+
+![Screenshot of template](http://contentful.github.io/extensions/assets/uiextensions-default-field-value.png)
+The UI Extension in the Contentful web app
+
+The default values are set through instance parameters when the extension is installed into a space:
+
```

Comment:
"The instance parameter is defined in the extension.json descriptor file as follows:"